### DISCUSSION TITLE
Busco equipo

### DISCUSSION THREAD

**Discussion by lODIN007l** [Launch-X-Latam/MisionFrontEnd#519] [Team Posts]
Hola .soy Wilson de Ecuador ,quisiera unirme o integrar un equipo ,he tenido experiencia con desarrollo web tanto front como back  con tecnologias como  ReactJS ,nEXTjs ,NodeJS ,Express ,Postgress ,MongoDBy demas bases y a su vez en el lado mobile con Flutter y un poco de React Native
Busco compañeros que desean trabajar en conjunto o unirme a un equipo ^^ ,aqui dejo mi email
lx06609@innovaccion.mx

**Comment by musicod3r** [comment]
Me aceptas en tu grupo? 
Soy Alvaro, de Colombia y me gustaría formar parte de un equipo, tengo conocimientos decentes de Node Js/Express Js, SQL y PHP...Y conocimientos básicos de React js/Next y Vue js
lx13359@innovaccion.mx, como aun no se por donde se ingresa a ese email, dejo otro sdahost@misena.edu.co

**Comment by CAMILOITT** [comment]
mi nombre es Camilo estoy interesado en hacer el grupo
correo: : lx11002@innovaccion.mx

**Comment by PepoGE** [comment]
Hola! Me llamo José Luis y estaría encantado de formar parte del equipo, ando cursando una Ingeniería en TI,  tengo conocimientos en FrontEnd, programación en Java y experiencia con UI, siempre dispuesto a aprender y ayudar con lo que pueda aportar :D 
Mi correo es: lx03228@innovaccion.mx

**Comment by dzarkV** [comment]
Hola, podría aportar al equipo. Mi correo es lx06352@innovaccion.mx

**Comment by charloo15** [comment]
Hola. Que tal? Me llamo Juan, soy de Argentina y si aún falta alguno, me gustaría sumarme al equipo. El mundo de la programación es nuevo para mi, pero tengo el animo de aprender y colaborar en lo que pueda =D Estoy estudiando Contabilidad y Administración de Empresas. Saludos. 
Mi correo lx04053@innovaccion.mx

**Comment by Jroot01** [comment]
Hagamos equipo!
